lib/ddtrace/contrib/redis/patcher.rb

```diff
@@ -59,7 +59,7 @@ def patch_redis_client
             end
 
             def initialize(*args)
-              pin = Datadog::Pin.new(SERVICE, app: 'redis', app_type: Datadog::Ext::AppTypes::CACHE)
+              pin = Datadog::Pin.new(SERVICE, app: 'redis', app_type: Datadog::Ext::AppTypes::DB)
```

Comment:
This should ultimately be the user to decide about. Redis can act as both, this is just a default value. Most of the "added value" of the patch here is: actually do use the value in the Pin and avoid redis spans being reported as "custom" (which is clearly not the case).